controllers/feeds/getOneFeedChat.js

```diff
@@ -119,13 +121,15 @@ const getOneFeedChatService = async (req, res) => {
     actor.karmaScores = roundingKarmaScore(user?.karma_score || 0);
 
     if (data?.results[0]?.anonimity) {
-      actor.data.username = `Anonymous ${data?.results[0]?.anon_user_info_emoji_name}`;
+      actor.data.username = `${data?.results?.[0]?.anon_user_info_color_name} ${data?.results[0]?.anon_user_info_emoji_name}`;
       actor.data.anon_user_info_emoji_name = data?.results[0]?.anon_user_info_emoji_name;
       actor.data.anon_user_info_emoji_code = data?.results[0]?.anon_user_info_emoji_code;
       actor.data.anon_user_info_color_name = data?.results[0]?.anon_user_info_color_name;
       actor.data.anon_user_info_color_code = data?.results[0]?.anon_user_info_color_code;
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This code block is modifying the `actor.data` object directly. It's generally a good practice to avoid mutating objects directly as it can lead to unexpected side effects. Instead, consider creating a new object with the updated properties.
```diff
      actor.data.username = `${data?.results?.[0]?.anon_user_info_color_name} ${data?.results[0]?.anon_user_info_emoji_name}`;
       actor.data.anon_user_info_emoji_name = data?.results[0]?.anon_user_info_emoji_name;
       actor.data.anon_user_info_emoji_code = data?.results[0]?.anon_user_info_emoji_code;
       actor.data.anon_user_info_color_name = data?.results[0]?.anon_user_info_color_name;
       actor.data.anon_user_info_color_code = data?.results[0]?.anon_user_info_color_code;
+      actor.data = {
+        ...actor.data,
+        username: `${data?.results?.[0]?.anon_user_info_color_name} ${data?.results[0]?.anon_user_info_emoji_name}`,
+        anon_user_info_emoji_name: data?.results[0]?.anon_user_info_emoji_name,
+        anon_user_info_emoji_code: data?.results[0]?.anon_user_info_emoji_code,
+        anon_user_info_color_name: data?.results[0]?.anon_user_info_color_name,
+        anon_user_info_color_code: data?.results[0]?.anon_user_info_color_code
+      }
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->